test/run/test.run.js

```diff
@@ -115,3 +116,23 @@ describe("fx-runner start", function () {
     });
   });
 });
+
+describe("concat binary arguments", function () {
+  it("binary arguments as string", function (done) {
```

Comment:
I find test names easier to read when they follow the mocha convention of reading like a sentence starting with *it*. In this case it could become `it("concats binary arguments from a string", ...)`